util/mkerr.pl

```diff
@@ -321,7 +321,7 @@ sub help
         s/[\n\s]*$//g;
 
         # Skip over recognized non-function declarations
-        next if /typedef\W/ or /DECLARE_STACK_OF/ or /TYPEDEF_.*_OF/;
+        next if /typedef\W/;
```

Comment:
You will need to partially restore the removed condition

Reply:
Why?  It is not used in openssl code, and mkerr is only "suppported" for use in openssl code.